apps/web/public/_nuxt-plenty/editor/blocksLists.json

```
@@ -966,6 +966,107 @@
       }
     ]
   },
+  "item-category": {
+    "category": "item-category",
+    "accessControl": [
+      "content",
+      "productCategory"
```

```suggestion
      "productCategory"
```
We don't need the grid on content pages. It wouldn't have any data.